set.mm

```diff
@@ -19,9 +19,9 @@ Dwight Megill (1950--2021).  For more information, visit
    been continuously enriched since then (list of contributors below). $)
 
 
-$(
+$( !
```

Comment:
I'm coming after the battle here, but I think maybe this exclamation mark was meant so that this comment is not considered a heading, and skipped when building the table of content.

Reply:
Yes, this is the followup to https://github.com/metamath/set.mm/pull/2485#issuecomment-1035675818